src/quantum_gates/utilities.py

```diff
@@ -9,4 +9,8 @@
     setup_backend,
     post_process_split,
     compute_Hellinger_distance as hellinger_distance,
+    create_random_quantum_circuit,
+    transpile_qiskit_circuit
 )
+
+from ._utility.circ_optimizator import Optimizator
```

Comment:
@paolodr98 Maybe we should give it the name "Optimizer" instead of "Optimizator", expect in the case that you are having the mismatch on purpose to avoid name-clashes. If in doubt, use tools like [Google Ngram](https://books.google.com/ngrams/graph?content=+Optimizator%2C+optimizer&year_start=1800&year_end=2022&corpus=en&smoothing=3) to find the more commonly used term.